.luacheckrc

```diff
@@ -2,3 +2,7 @@ std = "ngx_lua"
 files["spec"] = {
 	std = "+busted";
 }
+globals = {
+	"kong",
+}
+self = false
```

Comment:
Why is this warning turned off?

Reply:
It is a consequence of eliminating the "print the plugin name between square brackets" in the log. The `new_tracer` function only used `self` to obtain `self._name`. Since that was not needed any more, it would generate the warning "self is not being used".